packages/ds-medicare-gov/src/styles/components/_ChoiceList.scss

```diff
@@ -1,24 +0,0 @@
-.ds-c-choice-wrapper {
-  gap: 1rem;
-
-  &:has(.ds-c-choice--small) {
-    gap: 0.75rem;
-  }
-}
-
-.ds-c-choice:not(.ds-c-choice--small) + label,
-.ds-c-choice:not(.ds-c-choice--small) + .ds-c-label {
-  transform: translateY(0.125em);
-}
-
-.ds-c-choice[type='checkbox'] {
-  &:checked {
-    &::before {
-      border-radius: var(--choice__border-radius);
```

Comment:
This was fixed using the `inherit` value - meaning the border-radius of this pseudo selector will inherit the border-radius defined on its parent - a checked checkbox in this case.